api/src/cle/classe/classeService.ts

```diff
@@ -80,3 +81,26 @@ export const getEstimatedSeatsByEtablissement = async (etablissement: Etablissem
   const classes = await ClasseModel.find({ etablissementId: etablissement._id });
   return classes.reduce((classeNumberAcc: number, classe: ClasseDocument) => classeNumberAcc + classe.estimatedSeats, 0);
 };
+
+export const getClasseById = async (classeId, withPopulate = true) => {
+  let query = ClasseModel.findById(classeId);
+
+  if (withPopulate) {
```

Comment:
```suggestion
  if (!withPopulate) {
      return await ClasseModel.findById(classeId);
  }
```
que penses tu de ça ?